framework/src/abi/constants.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+export enum TransactionVerifyResult {
+	INVALID = 0,
+	PENDING = 1,
+	OK = 2,
+}
+
+export enum TransactionExecutionResult {
+	INVALID = 0,
+	FAIL = 1,
+	OK = 2,
+}
```

Comment:
FAIL=1 looks a bit odd, maybe 
```suggestion
export enum TransactionExecutionResult {
	INVALID = -1,
	FAIL = 0,
	OK = 1,
}
```

Reply:
Should we update verification result also to be similar?

Reply:
yeah true, I can take a few points we raised during PR reviews and create an issue

Reply:
Updated the both values to be similar